schema.json

```diff
@@ -1426,6 +1425,96 @@
     ]
   },
   {
+    "name": "Checkout",
+    "settings": [
+      {
+        "type": "heading",
+        "content": "Header"
+      },
+      {
+        "type": "checkbox",
+        "label": "Use custom background",
+        "description": "1000 x 400px recommended",
+        "force_reload": true,
+        "id": "uco_use_custom_background"
+      },
+      {
+        "type": "image",
+        "reference": "uco_use_custom_background",
+        "reference_default": false,
+        "force_reload": true,
+        "id": "uco_custom_background_image"
+      },
+      {
+        "type": "heading",
+        "content": "Logo"
+      },
+      {
+        "type": "select",

```

Comment:
Not sure that's what the designs are
<img width="285" alt="screen shot 2016-08-16 at 4 04 50 pm" src="https://cloud.githubusercontent.com/assets/8165665/17689460/32ddb4f6-63cb-11e6-8d9e-e768c99dfdbd.png">